### PR TITLE
Oneof accessor should return the field name that is actually set.

### PR DESCRIPTION
**Pull request by TeBoring**

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by TeBoring** [comment]
@stanley-cheung

File: php/src/Google/Protobuf/Internal/Message.php [hunk 1]
```diff
@@ -163,6 +163,14 @@ protected function writeOneof($number, $value)
         $oneof_field->setNumber($number);
     }
 
+    protected function whichOneof($oneof_name)
+    {
```
Comment (stanley-cheung) [inline php/src/Google/Protobuf/Internal/Message.php]: Sorry not familiar with the oneOf API in general. Is the `whichOneOf` naming consistent with other languages?

Comment (TeBoring) [inline php/src/Google/Protobuf/Internal/Message.php]: whichOneof is for implementation only. It's used to implement API in generated code for oneof.
For proto:
message Foo {
  oneof my_oneof {
    int32 a = 1;
    int32 b = 1;
  }
}
In generated code, message Foo will have the method "getMyOneof", which returns the name of the field that is currently set (e.g., a or b).

**Pull request merged**